lib/jsdom/living/postmessage.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+const isValidTargetOrigin = require("../utils").isValidTargetOrigin;
+
+module.exports = function (message, targetOrigin) {
+  let window = this;
+  let event;
+
+  if (!targetOrigin) {
```

Comment:
Should check `arguments.length < 2`, and then do `targetOrigin = String(targetOrigin)`. Passing `undefined` should be equivalent to passing `"undefined"`.